src/App.js

```diff
@@ -17,6 +17,7 @@ class App extends React.Component {
 
   componentDidMount() {
     this.props.dispatch(retrieveUserFromSession());
+    window.language = this.props.language; // for non react-intl localizations
```

Comment:
Why pollute `window`, when you have a perfectly cromulent [`config`](https://github.com/City-of-Helsinki/kerrokantasi-ui/blob/master/src/config.js) object?